public/locales/cs/data.json

```diff
@@ -5,10 +5,9 @@
   "footprint": "Ověření",
   "change-detection": "Porovnání",
   "project-card-progress-text": "{{progress, number}}% dokončeno",
-  "project-card-last-update": "{{date, datetime}}",
```

Comment:
we might need this, because this also happens during the build time. Need to take this from env or something in github pages.